ex_14_ch4.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: create constant
+WI_RESIDENT_TAX = 5.5
+
+#step2: get input from user for the total order amount
+order_subtotal = float(input("Enter the total order amount: $"))
+
+#step3: get input from user on the state
+state = input("Enter the state you live in, using an abbreviation (eg, MA): ")
+
+#step4: make tax calculation for WI resident
+tax_amount = WI_RESIDENT_TAX / 100 * order_subtotal
```

Comment:
Lines 19-20 only really apply to the `if` condition. So consider moving this code inside that branch.